### PR TITLE
@W-9359339 Feature/jest framework  data table wrapper

### PR DESCRIPTION
**Pull request by andrea-fernandez**
Link to Work Item: https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000Na97YAC/view

# Dev Changes

Increased test coverage of lwc `dataTableWrapper` to a 100%.

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
Refer to [Definition of Done](https://salesforce.quip.com/9P7hAOPHJJyU) to see any additional details for the items below:
- [x] Any net new LWC work has JEST test coverage 50% or above
- [x] Default Sa11y tests pass for all LWC components
~~- [ ] 🔒 Secure both Front-end (LWC) & back-end (Apex) as necessary~~
~~- [ ] 🔑 Grant users access in Permission Sets (Object, Field, Apex Class) as necessary~~
- [x] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: [W-0000000: Work Name]()
~~- [ ] Make sure that ACs are updated (if any gaps)~~
- [ ] **All acceptance criteria have been met**
    - [x] Developer
    - [ ] Code Reviewer
~~- [ ] Pull Request contains draft release notes~~
~~- [ ] Labels, help text, and customer facing messages are reviewed by Docs~~
~~- [ ] QE story level testing completed~~

### CODE REVIEW

File: force-app/main/default/lwc/dataTableWrapper/__tests__/dataTableWrapper.test.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ describe("c/dataTableWrapper", () => {
         component = null;
     });
 
-    it("FIXME should get code coverage for handleRowChange", () => {
+    it("should get code coverage for handleRowChange", () => {
```
Comment (seanpat09) [inline force-app/main/default/lwc/dataTableWrapper/__tests__/dataTableWrapper.test.js:19]: @a-contreras 
Can you change this to be more descriptive of what is expected from this test? Something like `"should dispatch draftvaluechange on row change"`

File: force-app/main/default/lwc/dataTableWrapper/__tests__/dataTableWrapper.test.js [hunk 2]
```diff
@@ -41,6 +41,13 @@ describe("c/dataTableWrapper", () => {
             draftValues[0]
         );
     });
+    it("should display the lightning-datatable", () => {
+        const checksTable = component.shadowRoot.querySelector("lightning-datatable");
+
+        expect(checksTable.columns).not.toBe(null);
+        expect(checksTable.records).not.toBe(null);
```
Comment (seanpat09) [inline force-app/main/default/lwc/dataTableWrapper/__tests__/dataTableWrapper.test.js:48]: @a-contreras
Checking that these are not null is a good start, but we also know that they should default to an empty array. For these `@api` attributes, we want to confirm:
* That they default to `[]`
* That they can be set

So you could do something like this:

```
expect(checksTable.columns).not.toBe(null);
expect(checksTable.columns).toBe([]);

let expectedColumns = ['Name', Phone'];
checksTable.columns = expectedColumns;
expect(checksTable.columns).toBe(expectedColumns);
```

**Review by seanpat09 — CHANGES_REQUESTED** [review]

File: force-app/main/default/lwc/dataTableWrapper/__tests__/dataTableWrapper.test.js (hunk 1 above)
Comment (a-contreras) [reply force-app/main/default/lwc/dataTableWrapper/__tests__/dataTableWrapper.test.js:19]: Sure!

File: force-app/main/default/lwc/dataTableWrapper/__tests__/dataTableWrapper.test.js (hunk 2 above)
Comment (a-contreras) [reply force-app/main/default/lwc/dataTableWrapper/__tests__/dataTableWrapper.test.js]: Done!

**Review by seanpat09 — APPROVED** [review]
@a-contreras nice work! thanks

**Pull request merged**